docs/code-snippets/custom-functions-snippets.yaml

```
@@ -49,4 +49,73 @@ CustomFunctions.ErrorCode:enum:
       // Return the results of the first and third parameter calculations
       // and a #NUM! error in place of the second result.
       return [[firstResult], [secondResult], [thirdResult]];
-      };
\ No newline at end of file
+      };
+CustomFunctions.Invocation:interface:
+  - |- 
+    /**
+    * Return the address of the cell that invoked the custom function. 
+    * @customfunction
+    * @param {number} first First parameter.
+    * @param {number} second Second parameter.
+    * @param {CustomFunctions.Invocation} invocation Invocation object. 
+    * @requiresAddress 
+    */
+    function getAddress(first, second, invocation) {
+      const address = invocation.address;
+      return address;
+    }
+CustomFunctions.Invocation#address:member:
+  - |- 
+    /**
+    * Return the address of the cell that invoked the custom function. 
+    * @customfunction
+    * @param {number} first First parameter.
+    * @param {number} second Second parameter.
+    * @param {CustomFunctions.Invocation} invocation Invocation object. 
+    * @requiresAddress 
+    */
+    function getAddress(first, second, invocation) {
+      const address = invocation.address;
+      return address;
+    }
+CustomFunctions.Invocation#isInValuePreview:member:
+  - |- 
+    /**
+    * Get the listing price for a house on the market for the given address.
+    * @customfunction
+    * @param address The address of the house.
+    * @param invocation Custom function handler.
+    * @returns The price of the house at the address.
+    */
+    function getHousePrice(address: string, invocation: CustomFunctions.Invocation): number {
+      // Check if this call is for formula value preview mode.
+      if (invocation.isInValuePreview) { 
+        // Avoid long-running expensive service calls. 
+        // Return a useable but fake number.
```

seems "useable" is less commonly used

```suggestion
        // Return a usable but fake number.
```